lldb/test/API/terminal/TestEditlineCompletions.py

```diff
@@ -0,0 +1,69 @@
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+from lldbsuite.test.lldbpexpect import PExpectTest
+
+
+class EditlineCompletionsTest(PExpectTest):
+    @skipIfAsan
+    @skipIfEditlineSupportMissing
+    @skipIfEditlineWideCharSupportMissing
```

Comment:
Are you sure that's necessary? I only see this decorator on a single test, and that one is sending squiggly worm emojis..

Reply:
Yeah, copy-paste :-)